darwin/exceptions.py

```diff
@@ -45,3 +45,6 @@ class ValidationError(Exception):
 
 class Unauthorized(Exception):
     pass
+
+class OutdatedDarwinJSONFormat(Exception):
+    pass
```

Comment:
Let's add a newline here (I think the formatter would do it, so if we haven't formatted the whole pr, this is our chance 😊)